tests/verifyDashboards_test.js

```diff
@@ -0,0 +1,26 @@
+Feature('Test Dashboards');
+const { dashboardPage } = inject();
+
+const dashboard = new DataTable(['page', 'content']);
+const mySQLContent = dashboardPage.fields.mySQLServiceSummaryContent;
```

Comment:
I think there is no need to declare variables here, just use them directly in `.add()` method

Reply:
This is solving of lint issues: https://github.com/percona/pmm-ui-tests/pull/20/commits/42ceeb69fec1b20f4f7c782ab0d857d71bd18079 :)

Reply:
Which issues?

Reply:
![image](https://user-images.githubusercontent.com/63704168/119450815-c2423500-bd34-11eb-9044-e591293525ed.png)

Reply:
So just split this line into 2 or 3 lines